### ISSUE TITLE
Header

### ISSUE THREAD

**Issue by den-kezlia** [Rudik585/NEw1#8]
Design — https://www.figma.com/design/DyHIhldVEgraYbsBDF2kcf/BGS-iBanking---Sample-Project-(Community)?node-id=6055-2436&t=Tdws5hjPSztfbIAv-1

Create a header with a simple navigation bar
Include the Logo which follows to Home page on Click
Icon links should have aria-label attributes for screen readers

**Comment by Rudik585** [comment]
done
